Makefile

```diff
@@ -24,7 +24,7 @@ sort: Makefile
 		echo "  https://github.com/dealii/publication-list#installing-bibtool" >&2 ; \
 		exit 1 ; \
 	fi ; \
-	for f in publications-*.bib ; do \
+	for f in $(shell find . -type f -regex ".*\.bib") ; do \
```

Comment:
This makes no sense -- the two should be exactly identical.

Is the problem you see (without the change here) reproducible?

Reply:
see below the latest answer.